File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        for x in range(len(nums)-1,0,-1):
            largest = x
            for i in range(x):
                if nums[largest] < nums[i]:
                    largest = i
            nums[largest],nums[x] = nums[x], nums[largest]

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int j;
        int len = nums.size();
      for(int i = 1; i< len;i++){
          int tmp = nums[i];
          for(j = i-1; j>=0 && nums[j] > tmp;j--){
                  nums[j+1] = nums[j];
          }
          nums[j+1] = tmp;
      }
      
    }
};